frontend/src/modules/projectinfo/components/ProjectInfo.js

```diff
@@ -54,11 +55,21 @@ export class ProjectInfoBase extends React.Component<Props, State> {
 
         return <div className="project-info">
             <div className="button" onClick={ this.toggleVisibility }>
-                <span className="fa fa-info"></span>
+              <Localized
+                  id="projectinfo-ProjectInfo--project-info"
+                  glyph={
+                      <span className="fa fa-info"></span>
+                  }
+              >
+              </Localized>
```

Comment:
There is no string content there, only an icon, so there is no need to translate it. Please revert this part and remove then entry in the .ftl file.